pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -17,8 +17,8 @@
     GenericParameters,
     GenericMaster,
     GenericJob as GenericJobCore,
-    deprecate,
 )
+from pyiron_snippets.deprecate import deprecate
```

Comment:
The import feels a bit redundant `from pyiron_snippets.deprecate import deprecate`. Can we change it to `from pyiron_snippets import deprecate` ?

Reply:
Certainly not on this version of snippets. Personally, to maintain the idea that snippets is exceedingly lightweight and that all its pieces are independent, I prefer to leave its `__init__` empty. But I'm not totally closed to it so feel free to open an issue on snippets and see what the others think